packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts

```diff
@@ -54,12 +54,66 @@ const cardStyles: ComponentSlotStylesPrepared<CardStylesProps, CardVariables> =
         },
       }),
 
+      ...(p.elevated && {
+        boxShadow: v.elevatedBoxShadow,
+        ':hover': {
+          boxShadow: v.elevatedBoxShadowHover,
+        },
+
+        ...(p.actionable && {
+          ':focus-visible': {
```

Comment:
Why are we not using the `getBorderFocusStyles` here? We can invoke it with different variables based on the need

Reply:
Good catch, thank you